action.yml

```diff
@@ -0,0 +1,69 @@
+---
+name: mcaf-mcvs-golang-action
+description: |
+  The Mission Critical Acceleration Framework (MCAF)
+  Mission Critical Vulnerability Scanner (MCVS) Golang action.
+inputs:
+  golang-version:
+    description: |
+      The Golang version that has to be installed and used.
+    default: 1.21.5
+    required: true
+  golangci-lint-version:
+    description: |
+      The Golangci-lint version that has to be installed and used.
+    default: v1.55.2
+    required: true
+  golang-unit-tests-exclusions:
+    description: |
+      The Golang paths that should be excluded from unit testing.
+    default: ' '
+runs:
+  using: 'composite'
+  steps:
+    #
+    # install a certain golang version
+    #
+    - uses: actions/setup-go@v4.0.0
```

Comment:
I think the latest is actually v5 :) 
https://github.com/actions/setup-go

Reply:
Dependa-bot should take care of that, but good point